README.md

```diff
@@ -202,28 +200,28 @@ const service = sdk.service(function(err, service) {
 };
 ```
 
-## [Business Logic framework](#business-logic-framework)
+## [FlexFunctions](#flex-functions)
 
-The Business Logic framework can be accessed via the sdk's `businessLogic` property.
+The FlexFunctions framework is used to execute functions invoked by `hooks` or `endpoints`.  It can be accessed via the sdk's `functions` property.
 
 ```
-const dataLink = sdk.businessLogic;
+const functions = flex.functions;
 ```
 
-### Registering Business Logic Handlers
+### Registering FlexFunction Handlers
 
-In order to register a business logic handler, you define that handler and give it a name by using the `register` method of the `businessLogic` framework:
+In order to register a FlexFunction handler, you define that handler and give it a name by using the `register` method of the `functions` framework:
```

Comment:
Should be either:

'use the serviceObject method of the functions object:'

or

'use the serviceObject method of the FlexFunctions framework:'